src/main/java/com/aws/greengrass/util/Exec.java

```diff
@@ -219,7 +206,7 @@ private static void ensurePresent(String... fns) {
 
     private static void addPathEntries(String path) {
         if (path != null && path.length() > 0) {
-            for (String f : path.split("[ :,] *")) {
+            for (String f : path.split("[" + PATH_SEP + ",] *")) {
```

Comment:
Nice catch it was splitting by space...

Reply:
I think that _may_ have been intentional @JamesGosling, but it doesn't work for Windows where space in paths is fine.